lib/src/components/mixins/has_game_ref.dart

```diff
@@ -1,5 +1,5 @@
 import '../../game/game.dart';
 
 mixin HasGameRef<T extends Game> {
-  T gameRef;
+  T? gameRef;
```

Comment:
this should probably be late but I had some trouble with our tests

Reply:
I agree! Could you share which was the trouble you faced?

Reply:
test break because it is not setup properly in some of our test scenarios